docs/service-mesh/14-02-no-access.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Can't access Console UI or other endpoints
+type: Troubleshooting
+---
+
+A response of code `503` error when trying to access the Console UI or any other endpoint in Kyma can be caused by a configuration error in the Istio Ingress gateway. As a result, the endpoint you call is not exposed.
```

Comment:
```suggestion
The `503` status code received when you try to access the Console UI or any other endpoint in Kyma indicates a configuration error in the Istio Ingress gateway. As a result, the endpoint you call is not exposed.
```

Reply:
Well, unfortunately, this is not binary. A 503 error CAN be caused by the described gateway error but there are other things that can cause it too. I'll apply your suggestion but I'll edit it a bit to indicate that this is not ALWAY the case,